### PR TITLE
EL-1452: Remove feature flag & redundant code

### PR DESCRIPTION
**Pull request by MazOneTwoOne**
## What does this pull request do?

- remove old templates
- update .ene.example
- get rid of conditionals
- update CSS
- removed behave tests as part of the CircleCi `config.yml` (will need to tidy up `cla-end-to-end-tests` at a later time). 

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: fala/apps/adviser/tests/test_search_view_function.py [hunk 1]
```diff
@@ -114,18 +110,11 @@ class ResearchBannerTest(SimpleTestCase):
 
     research_message = "Help improve this legal adviser search"
 
-    @override_settings(FEATURE_FLAG_SURVEY_MONKEY=True)
-    def test_research_banner_present_when_flag_set(self):
+    def test_research_banner_present(self):
```
Comment (starswan) [inline fala/apps/adviser/tests/test_search_view_function.py:113]: ditto for survey monkey flag - need to keep

File: fala/templates/footer.html [hunk 2]
```diff
@@ -46,7 +46,7 @@ <h2 class="govuk-visually-hidden">Support links</h2>
         <a class="govuk-footer__link" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence v3.0</a>, except where otherwise stated
       </span>
     </div>
-    <div class="govuk-footer__meta-item">
```
Comment (starswan) [inline fala/templates/footer.html:49]: I think we should be keeping this?

File: .env.example [hunk 3]
```diff
@@ -4,8 +4,6 @@ DEBUG = True
 ALLOWED_HOSTS = ["localhost", '127.0.0.1']
 LAALAA_API_HOST = "https://laa-legal-adviser-api-staging.apps.live-1.cloud-platform.service.justice.gov.uk"
 ENVIRONMENT = "dev"
-FEATURE_FLAG_NO_MAP = False
-FEATURE_FLAG_SURVEY_MONKEY = False
```
[the remark this answers is not in the document]
Comment (MazOneTwoOne) [reply .env.example:8]: @starswan whoops.... my eyes were playing tricks on me and could have sworn, we also wanted to remove this flag. Will amend now.

File: fala/templates/footer.html (hunk 2 above)
Comment (MazOneTwoOne) [reply fala/templates/footer.html:49]: @starswan yes you're right. I think I changed the CSS to see the difference and then forgot to put back.

**Review by psweeting1 — APPROVED** [review]
LGTM

**Pull request merged**